### ISSUE TITLE
Initial thoughts on direction

### ISSUE THREAD

**Issue by Jimbo-shack** [fedorSulitskiy/ParliamentAccountabilityDocs#1]
### What is our overarching goal?

- [ ] News - Live?
- [ ] News - website?
- [ ] Government - information about KPIs?
- [ ] Private - for individuals and companies to consume?

### What is the quickest, simplest, easiest way to test the need?

- [ ] Visualising the existing data with a focus on?
- [ ] Do we need to combine parlement data with other data?
- [ ] What is the cutting edge?

### Techstack

- [ ] Potentially useful starting point; https://github.com/haydenbleasel/next-forge

**Comment by fedorSulitskiy** [comment]
# Looking into What Matters

The way I see it we need to strike the niche of information that is not covered by normal media and/or use our data presentation skills to create a "knowledge hub" national decisions and statistics. What do I mean by that?

------------------------------------------------------------------------------------------------------------------------------------

## Looking into what the Government says...

People don't tend to know what is being discussed in the Houses of Parliament, and this creates an aura of limited accountability. Therefore, I see it as desirable to look into what is being said in the Commons and the Lords, _AND_ summarise _AND_ provide context. Hence, we can develop a UI where it would list the current debates, their transcripts, and employ LLMs to create digestible summaries, and extract keywords out of those debates. For those keywords we can employ an internet search to provide context (can start with like Wikipedia API, and expand capability in the future)

**BUT** we live in a ~~society~~ in a parliamentary first-past-the-post democracy, meaning essentially the Government controls the Commons, so its debates aren't as important as one might assume. The Lords are more interesting. And what is MOST interesting is the _Laws_ passed. Hence we can think of a UI, a feed, of new laws, amendments and statutory instruments being passed (if available by any API). Likewise, provide summaries, and context using LLMs. For Acts of Parliament show the progress stages like on `.gov` website; First Reading, Second Reading, Royal Ascent etc.

Furthermore, for _even wider_ context, I think we should create a repository of Acts in progress. All of the legislation that is currently going through the government pipeline, with all appropriate metrics like time in backlog, number of readings etc. This would show give an easy way of scrutinizing what the hell is going on.

Finally, and I this one is a more complex task as it relies on semantic analysis more than the rest, create a UI which would track campaign promise completion, by matching said promises with government actions. In the most basic form, the manifesto can be manually dissected into "promises" that can be semantically compared to aforementioned debates, and legislation pipeline to link them together. There is more value in back-filling this system for past governments as as far back as the data goes.

## TLDR Proposed Features;

-  Houses of Parliament Debate interface (both chambers)
   - Transcripts
   - Summaries
   - Context
- Legislation Feed
   - Acts of Parliament timeline & feed
   - Statutory Instruments feed & full context
   - Amendments & full context
- Acts of Parliament backlog interface
   - All Acts of Parliament progress in backlog
- Campaign Accountability Action Aggregation
   - Match Campaign (manifesto) promises to government action
   
------------------------------------------------------------------------------------------------------------------------------------

## Looking into National Statistics

Create a nice, interactive interface with dynamic recalculation of quality of life & economic statistics. I envision interactive charts straight from the Office of National Statistics and other data collectors which can be superimposed on each other and measured for various time periods. This can be for such metrics as:

- Median & Average Wages
- Inflation (obviously)
- Purchasing Power over time
- GDP
- GDP per capita (PPE)
- Crime statistics & representation
- etc.

Having an accessible and reliable hub of such data would be pretty nice! As experience shows in today's day and age, it's not enough to have data accessible somewhere, where there is value is to have this data in one place, in an organised and superimposable way.
